helm/ph-ee-engine/vouchers/values.yaml

```diff
@@ -67,7 +67,7 @@ ingress:
   #  - secretName: sandbox-secret
   #    hosts:
   #      - chart-example.local      
-
+paymentadvice: true
```

Comment:
can you keep this disabled here and enable it from fyn-sit chart so that the base chart is not affected by load testing

Reply:
idea is to keep the default behaviorr in the charts, it will be easy to do releases and other integration tests. Special cases can be custom

Reply:
with paymentAdvice as true it will follow the default behaviour.